crates/pallet-feeds/Cargo.toml

```diff
@@ -36,4 +36,4 @@ std = [
   "frame-system/std",
   "log/std",
 ]
-try-runtime = ["frame-support/try-runtime"]
+try-runtime = ["frame-support/try-runtime"]
```

Comment:
Nitpick, but I already mentioned it, please configure IDE to add new lines to the end of the file automatically and definitely not to remove those that were already in place.

When you decide to print the file with `cat Cargo.toml` without new line your terminal will (likely, but not necessarily) be a bit messed up due to lack of the new line.